index.js

```diff
@@ -82,59 +67,89 @@ class AddLambdaInsights {
     if (typeof value === 'number') {
       return value;
     } else {
-      throw new Error(`LambdaInsightsVersion must be a number.`);
+      throw new Error('lambdaInsightsVersion version must be a number.');
     }
   }
 
+  /**
+   * Generates a valid Lambda Insights Layer ARN for your Region
+   * @param  {number} version Value to check
+   * @return {string} Lambda Insights Layer ARN
+   */
+  async generateLayerARN(version) {
```

Comment:
Should we also allow overriding the account id? I'm thinking of a case when a new version is released in an opt-in region but it hasn't been updated here.

Reply:
Very nice idea, let's make a complete new Feature Request out of it.